src/header/header.scss

```diff
@@ -252,20 +204,21 @@
       overflow: hidden;
       text-overflow: ellipsis;
     }
+  }
 
-    .iui-description {
-      height: $iui-baseline * 2;
-      @media (prefers-reduced-motion: no-preference) {
-        transition: all $iui-speed-fast ease;
-      }
-      @include themed {
-        opacity: t(iui-opacity-3);
-      }
+  .iui-description {
+    height: $iui-baseline * 2;
+    @media (prefers-reduced-motion: no-preference) {
+      transition: all $iui-speed-fast ease;
```

Comment:
is `all` really needed here?

Reply:
Changed it to just `height` & `ease-out` (matching the rest of the header transitions).  Now the animation is a bit smoother than before.